### ISSUE TITLE
Add an option for cluster installation waiting 

### ISSUE THREAD

**Issue by apodhrad** [openshift-online/ocm-cli#197]
When automation comes into play, we must deal with various waiting conditions.
One of them is waiting until the cluster is installed and ready to use.
Some teams implement it as follows
https://gitlab.cee.redhat.com/mk-bin-packing/mk-performance-tests/-/blob/master/scripts/osd-provision.sh#L227

but it would be great if we had such implementation directly in the `ocm-cli`, wdyt?

**Comment by cben** [comment]
Could also copy log-streaming from rosa `--watch` (https://github.com/openshift/rosa/pull/91).
@vkareh thoughts?

**Comment by vkareh** [comment]
Yes, that would be useful, but might be a problem for automation to deal with the output stream.

A less verbose alternative is to have a `ocm describe cluster --watch` that instead of streaming logs, just checks whether the `cluster.State()` (or any other attribute in the output) has changed and outputs the latest. Once it's `Ready`, it exits. This would be more in line with how `openshift-client` does it when you `oc get foo --watch`
